inference-engine/tests/unit/inference_engine/ie_compilation_context_test.cpp

```diff
@@ -131,16 +131,21 @@ static std::shared_ptr<ngraph::Function> create_simple_function() {
     // Create opset6::Parameter operation with static shape
     auto data = std::make_shared<ngraph::opset6::Parameter>(ngraph::element::i8, ngraph::Shape{3, 1, 2});
     data->set_friendly_name("Parameter");
+    data->get_output_tensor(0).set_names({"parameter"});
```

Comment:
Without 'set_names' these tests are failed, right? Will IR .xml file always have names for each tensor? Otherwise it is quite dangerous for caching feature (compilation context will be always different after read_network of same IR)

Reply:
IRs already have tensor names for all tensors. It is a required attribute.

Reply:
But if network is generated by code (e.g. integration scenario), we will have cache invalidated each time.
So, integration scenarios should set tensor names as well as operation ones.